docs/faq.rst

```diff
@@ -48,6 +48,44 @@ and platform/software dependency. Combining Borg with the mechanisms provided
 by the platform (snapshots, hypervisor features) will be the best approach
 to start tackling them.
 
+How can I decrease the size of disk image backups?
+--------------------------------------------------
+
+Full disk images are as large as the full disk when uncompressed and don't get much
+smaller post-deduplication after heavy use. This is because virtually all file systems
```

Comment:
guess one can't tell the dedup reduction that general (the user might have tons of duplicate data **in** existing files).
but it gets clearer what you mean in the following text, so maybe fix phrasing above like "and might not get...".